src/stories/Forms/TextArea/index.tsx

```diff
@@ -1,85 +1,48 @@
 import React from 'react';
 import styled from 'styled-components';
+import { GlobalInputProps } from '../../../shared/types';
 
-import InputWrapper from '../InputWrapper';
-import Label from '../Label';
 interface WrapperProps {
   hasError?: boolean;
 }
 
-const Wrapper = styled(InputWrapper)<WrapperProps>`
+const Wrapper = styled.div<WrapperProps>`
   textarea {
     display: block;
     width: 100%;
-    height: var(--spacing-48);
     height: 120px;
     padding: var(--spacing-16);
-    border: var(--border);
+    border: ${(p) =>
+      p.hasError ? 'var(--border-error)' : 'var(--border-primary)'};
     border-radius: var(--border-radius);
     font-family: var(--font-family-primary);
     resize: vertical;
-    // If there is an error, apply the error border
-    ${(p) => p.hasError && `border-color: var(--color-alertRed);`}
   }
 `;
 
-export interface TextAreaProps {
-  id: string;
-  label?: string;
-  value?: string;
-  onChange?: (event: React.ChangeEvent<HTMLTextAreaElement>) => void; // eslint-disable-line no-unused-vars
-  isRequired?: boolean;
-  alertLabel?: string;
-  helpLink?: string;
-  helpLabel?: string;
-  className?: string;
+export interface TextAreaProps extends GlobalInputProps {
+  /** Additional props here */
```

Comment:
If there is no extra props do we need a TextAreaProps or can we just use GlobalInputProps?